src/Build/BuildCheck/Infrastructure/EditorConfig/README.md

```diff
@@ -0,0 +1,98 @@
+﻿# EditorConfigParser
```

Comment:
@rainersigwald the documentation is updated could you please double check if there is anything else you would like to have documented?

Reply:
Resolving for now. In case clarifications will be needed the doc will be updated. Thank you